### PR TITLE
New HOS coat, warden jacket sprites

### PR DESCRIPTION
**Pull request by Arokha**
Now you look more like a buff sec person than an edgelord.

dreamseeker_2020-05-14_20-06-43

From Eris.

### CODE REVIEW

File: code/modules/clothing/suits/armor_vr.dm [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
 /obj/item/clothing/suit/armor/combat/USDF
 	name = "marine body armor"
-	desc = "When I joined the Corps, we didn't have any fancy-schmanzy armor. We had sticks! Two sticks, and a rock for the whole platoon�and we had to <i>share</i> the rock!"
+	desc = "When I joined the Corps, we didn't have any fancy-schmanzy armor. We had sticks! Two sticks, and a rock for the whole platoon�and we had to <i>share</i> the rock!"
```
Comment (Novacat) [inline code/modules/clothing/suits/armor_vr.dm:62]: VSCode did a fucky wucky

**Comment by KillianKirilenko** [comment]
Not gonna lie, that grey jacket gives me some big Original Deus Ex vibes.

Could probably do with some trademark red and gold bits... maybe I'll take a crack at a recolor-with-added-trim at somepoint.

File: code/modules/clothing/suits/armor_vr.dm (hunk 1 above)
Comment (Arokha) [reply code/modules/clothing/suits/armor_vr.dm:62]: That's not vscode, that character was a magical MS word long hyphen before. I'll fix it with a normal hyphen.

**Comment by Arokha** [comment]
`How about having new items instead of replacements, and then add the new items to the HoS/Warden lockers so they can choose?`

They already start with like 4 coats, I'd rather not add *more*. The HoS currently starts with 5 suits + the one they start with. Warden starts with 4+1.

**Comment by KillianKirilenko** [comment]
Repurpose gunbox code maybe? Have a case that you interact with to spawn your preferred coat, throw a backup in there or in cargo, boom, done.

**Comment by Arokha** [comment]
Lynx is going to mrowl it, cus I'm lazy.

**Comment by Arokha** [comment]
grumble

**Comment by KasparoVy** [comment]
Oh my God JC, a |BOMB|

**Comment by SingoDingo** [comment]
I'm kinda digging the Deus Ex vibes the HoS coat has, but I'll be honest I prefer the current warden jacket.

**Pull request merged**